dubbo-plugin/dubbo-reactive/src/main/java/org/apache/dubbo/reactive/AbstractTripleReactorSubscriber.java

```diff
@@ -53,7 +53,7 @@ public void subscribe(final CallStreamObserver<T> downstream) {
         if (downstream == null) {
             throw new NullPointerException();
         }
-        if (this.downstream == null && SUBSCRIBED.compareAndSet(false, true)) {
+        if (SUBSCRIBED.compareAndSet(false, true)) {
```

Comment:
Please help optimize the code style. Thanks.

Reply:
I ran spotless:apply again. Do you mean the hard coding of true and false?